apps/backoffice-v2/src/pages/Entity/components/Case/hooks/usePendingRevisionEvents/usePendingRevisionEvents.tsx

```diff
@@ -0,0 +1,60 @@
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+import { useCallback, useMemo } from 'react';
+import { calculateAllWorkflowEvents } from '@/pages/Entity/components/Case/hooks/usePendingRevisionEvents/utils/calculate-pending-workflow-events';
+import { CommonWorkflowEvent } from '@ballerine/common';
+import { checkIsKybExampleVariant } from '@/lib/blocks/variants/variant-checkers';
+import { useRevisionCaseMutation } from '@/domains/workflows/hooks/mutations/useRevisionCaseMutation/useRevisionCaseMutation';
+import { IPendingEvent } from './interfaces';
+
+function composeUniqueWorkflowEvents(
```

Comment:
Please ensure consistency for syntax used for functions. Prefer arrow functions over function expressions.